src/js/consts/dataFormat/dataLoading.jsx

```diff
@@ -1,90 +1,50 @@
-import {
-  colorLineChart,
-  colorBarChart,
-  colorHorizontalBarChart,
-  doughnutChart,
-  graphMultipleChart0,
-  graphMultipleChart1,
-  graphMultipleChart2,
-} from './chartColors';
+/* global _ */
```

Comment:
The same changes were pushed to https://github.com/openboxes/openboxes/pull/1487/files